js/src/common/components/Navigation.js

```diff
@@ -53,6 +53,7 @@ export default class Navigation extends Component {
         e.preventDefault();
         history.back();
       },
+      'aria-label': app.translator.trans('core.lib.close_button'),
```

Comment:
I think it would be more appropriate to have it as "Back" rather than "Close".

Reply:
Agreed @davwheat , my mistake.  Just pushed that change.